sklearn/decomposition/_kernel_pca.py

```diff
@@ -69,16 +97,25 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         Maximum number of iterations for arpack.
         If None, optimal value will be chosen by arpack.
 
+    iterated_power : int >= 0, or 'auto', (default 'auto')
```

Comment:
for consistency with the rest of the code base, call this `n_iter` instead?

what does auto do?

Maybe we can even not expose this as a first version?

Reply:
I copied the one in https://github.com/scikit-learn/scikit-learn/blob/95d4f0841/sklearn/decomposition/_pca.py#L186